mdata/store_cassandra.go

```diff
@@ -435,7 +446,12 @@ func (c *CassandraStore) SearchTable(key, table string, start, end uint32) ([]ch
 	results := make(chan outcome, numQueries)
 	for i := range crrs {
 		crrs[i].out = results
-		c.readQueue <- crrs[i]
+		select {
+		case c.readQueue <- crrs[i]:
+		default:
+			cassReadQueueFull.Inc()
+			numQueries--
```

Comment:
Actually that's why `numQueries` is decreased, so that loop on #459 does not wait for this result